eng/Packages.Data.props

```diff
@@ -0,0 +1,103 @@
+    <PackageReference Update="Microsoft.ServiceFabric" Version="6.4.624" />
+    <PackageReference Update="Microsoft.SourceLink.GitHub" Version="1.0.0-beta2-18618-05" />
+    <PackageReference Update="Microsoft.VisualStudio.TextTemplating.14.0" Version="14.3.25407" />
+    <PackageReference Update="Moq" Version="4.10.1" />
+    <PackageReference Update="MSTest.TestAdapter" Version="1.3.2" />
+    <PackageReference Update="MSTest.TestFramework" Version="1.3.2" />
+    <PackageReference Update="Newtonsoft.Json" Version="10.0.3" />
+    <PackageReference Update="NSubstitute" Version="3.1.0" />
+    <PackageReference Update="nunit" Version="3.12.0" />
+    <PackageReference Update="NUnit3TestAdapter" Version="3.13.0" />
+    <PackageReference Update="Polly" Version="7.1.0" />
+    <PackageReference Update="Portable.BouncyCastle" Version="1.8.5" />
+    <PackageReference Update="PublicApiGenerator" Version="8.1.0" />
+    <PackageReference Update="System.Buffers" Version="4.5.0" />
+    <PackageReference Update="System.Collections.Concurrent" Version="4.3.0" />
+    <PackageReference Update="System.Collections" Version="4.3.0" />
+    <PackageReference Update="System.Data.SqlClient" Version="4.3.1" />
+    <PackageReference Update="System.Diagnostics.DiagnosticSource" Version="4.5.1" />
+    <PackageReference Update="System.Diagnostics.Tools" Version="4.3.0" />
+    <PackageReference Update="System.Diagnostics.TraceSource" Version="4.3.0" />
+    <PackageReference Update="System.Globalization" Version="4.3.0" />
+    <PackageReference Update="System.IdentityModel.Tokens.Jwt" Version="[5.4.0, 6.0.0)" />
+    <PackageReference Update="System.IO.Compression" Version="4.3.0" />
+    <PackageReference Update="System.Linq" Version="4.3.0" />
+    <PackageReference Update="System.Memory" Version="4.5.2" />
+    <PackageReference Update="System.Net.Http" Version="4.3.4" />
+    <PackageReference Update="System.Net.WebSockets.Client" Version="4.3.2" />
+    <PackageReference Update="System.Numerics.Vectors" Version="4.5.0" />
+    <PackageReference Update="System.Reflection.TypeExtensions" Version="[4.5.1, 4.9.0)" />
+    <PackageReference Update="System.Runtime.CompilerServices.Unsafe" Version="4.5.2" />
+    <PackageReference Update="System.Runtime.Extensions" Version="4.3.0" />
+    <PackageReference Update="System.Runtime.InteropServices" Version="4.3.0" />
+    <PackageReference Update="System.Runtime.Serialization.Primitives" Version="4.3.0" />
+    <PackageReference Update="System.Runtime" Version="4.3.0" />
+    <PackageReference Update="System.Security.Cryptography.Algorithms" Version="4.3.0" />
+    <PackageReference Update="System.Security.Cryptography.Cng" Version="4.3.0" />
+    <PackageReference Update="System.Security.Cryptography.Primitives" Version="4.3.0" />
+    <PackageReference Update="System.Text.Json" Version="4.6.0-preview6.19259.10" />
+    <PackageReference Update="System.Threading.Tasks.Extensions" Version="4.5.1" />
```

Comment:
I believe this is what is causing your test failure. You need to bump System.Threading.Tasks.Extensions to 4.5.2.